editor/src/components/editor/store/editor-state.ts

```diff
@@ -2545,26 +2547,23 @@ function getElementWarningsInner(
 
     // Identify if this element looks to be trying to position itself with "pins", but
     // the parent element isn't appropriately configured.
-    let absoluteWithUnpositionedParent: boolean = false
-    if (
-      elementMetadata.specialSizeMeasurements.position === 'absolute' &&
+    const isParentGroupLike = treatElementAsContentAffecting(
+      rootMetadata,
+      allElementProps,
+      EP.parentPath(elementMetadata.elementPath),
+    )
+
+    const isParentConfiguredForPins =
+      MetadataUtils.isPositionAbsolute(elementMetadata) &&
       !elementMetadata.specialSizeMeasurements.immediateParentProvidesLayout
-    ) {
-      absoluteWithUnpositionedParent = true
-    }
+    const absoluteWithUnpositionedParent = isParentConfiguredForPins && isParentGroupLike === false
```

Comment:
fixed in https://github.com/concrete-utopia/utopia/pull/3400/commits/dfbf64598aa967c58e9d53b828db3440a28578ca

Though just for the note, I think === false is way easier to read than `!` because it's simply "bigger" on screen, and avoids having to mentally emulate the `!` operator